src/stravalib/tests/unit/test_model.py

```diff
@@ -347,7 +343,7 @@ def test_subscription_update_deser(self):
             "aspect_type": "create",
             "event_time": 1297286541,
         }
-        subupd = model.SubscriptionUpdate.deserialize(d)
+        subupd = model.SubscriptionUpdate.parse_obj(d)
```

Comment:
based on my question about - i'll fix or remove this change accordingly.

Reply:
This type of call to `deserialize()` should still be supported in v1 because users may be doing this. For v2, we should indeed replace it by `parse_obj()` (or its Pydantic2 equivalent).

Reply:
ok great. i've reverted this change!